sympy/stats/crv_types.py

```diff
@@ -163,6 +163,8 @@ def rv(symbol, cls, args):
 class ArcsinDistribution(SingleContinuousDistribution):
     _argnames = ('a', 'b')
 
+    set = Interval(0, 1)
```

Comment:
I think it should be `set = Interval(a, b)`as given [here](https://en.wikipedia.org/wiki/Arcsine_distribution#Generalization).

Reply:
Thank you!